core/systems/promotion/landing_page.ex

```diff
@@ -144,11 +141,13 @@ defmodule Systems.Promotion.LandingPage do
 
         <Text.title2 margin=""><%= dgettext("eyra-promotion", "expectations.public.label") %></Text.title2>
         <.spacing value="M" />
-        <Text.body_large><%= @vm.expectations %></Text.body_large>
+        <%!-- These fields use the raw() helper to make sure contents from the wysiwyg are rendered --%>
+        <Text.body_large><%= raw(@vm.expectations) %></Text.body_large>
         <.spacing value="M" />
         <Text.title2 margin=""><%= dgettext("eyra-promotion", "description.public.label") %></Text.title2>
         <.spacing value="M" />
-        <Text.body_large><%= @vm.description %></Text.body_large>
+        <%!-- These fields use the raw() helper to make sure contents from the wysiwyg are rendered --%>
```

Comment:
If you feel a need to add comments in the html/code, maybe add an extra layer (function) with self explanatory name. Consider: 

```
def render_wysiwyg(wysiwyg_text) do
   raw(wysiwyg_text)
end
```

This function could be placed in `WysiwygAreaHelpers`